hypervisor/arch/x86/configs/vacpi.c

```diff
@@ -121,11 +155,14 @@ void build_vacpi(struct acrn_vm *vm)
 	(void)copy_to_gpa(vm, xsdt, ACPI_XSDT_ADDR, sizeof(struct acpi_table_header));
 	xsdt = (struct acpi_table_xsdt *)gpa2hva(vm, ACPI_XSDT_ADDR);
 	stac();
-	xsdt->table_offset_entry[0] = ACPI_FADT_ADDR;
-	xsdt->table_offset_entry[1] = ACPI_MCFG_ADDR;
-	xsdt->table_offset_entry[2] = ACPI_MADT_ADDR;
+	xsdt->table_offset_entry[table_entry++] = ACPI_FADT_ADDR;
+	xsdt->table_offset_entry[table_entry++] = ACPI_MCFG_ADDR;
+	xsdt->table_offset_entry[table_entry++] = ACPI_MADT_ADDR;
```

Comment:
Why we change these code?

Reply:
Because the ACPI Table entry varied when there's a  MMIO device

Reply:
xsdt->table_offset_entry[0] is no longer ACPI_FADT_ADDR?  I am lost

Reply:
xsdt->table_offset_entry[0] is for ACPI_FADT_ADDR. and xsdt->table_offset_entry[0] is for ACPI_TPM2_ADDR only when we PT the TPM2 device to this pre-launched VM.

Reply:
No need for this change